projects/client/RabbitMQ.Client/src/client/impl/AutorecoveringConnection.cs

```diff
@@ -63,339 +60,233 @@ internal sealed class AutorecoveringConnection : IConnection
-            }
-            remove
-            {
-                lock (m_eventLock)
-                {
-                    m_recordedBlockedEventHandlers -= value;
-                    m_delegate.ConnectionBlocked -= value;
-                }
-            }
-        }
+        public event EventHandler<ShutdownEventArgs> ConnectionShutdown;
 
-        public event EventHandler<ShutdownEventArgs> ConnectionShutdown
-        {
-            add
-            {
-                lock (m_eventLock)
-                {
-                    m_recordedShutdownEventHandlers += value;
-                    m_delegate.ConnectionShutdown += value;
-                }
-            }
-            remove
-            {
-                lock (m_eventLock)
-                {
-                    m_recordedShutdownEventHandlers -= value;
-                    m_delegate.ConnectionShutdown -= value;
-                }
-            }
-        }
+        public event EventHandler<EventArgs> ConnectionUnblocked;
 
-        public event EventHandler<EventArgs> ConnectionUnblocked
-        {
-            add
-            {
-                lock (m_eventLock)
-                {
-                    m_recordedUnblockedEventHandlers += value;
-                    m_delegate.ConnectionUnblocked += value;
-                }
-            }
-            remove
-            {
-                lock (m_eventLock)
-                {
-                    m_recordedUnblockedEventHandlers -= value;
-                    m_delegate.ConnectionUnblocked -= value;
-                }
-            }
-        }
-
-        public event EventHandler<ConsumerTagChangedAfterRecoveryEventArgs> ConsumerTagChangeAfterRecovery
-        {
-            add
-            {
-                lock (m_eventLock)
-                {
-                    m_consumerTagChange += value;
-                }
-            }
-            remove
-            {
-                lock (m_eventLock)
-                {
-                    m_consumerTagChange -= value;
-                }
-            }
-        }
+        public event EventHandler<ConsumerTagChangedAfterRecoveryEventArgs> ConsumerTagChangeAfterRecovery;
 
-        public event EventHandler<QueueNameChangedAfterRecoveryEventArgs> QueueNameChangeAfterRecovery
-        {
-            add
-            {
-                lock (m_eventLock)
-                {
-                    m_queueNameChange += value;
-                }
-            }
-            remove
-            {
-                lock (m_eventLock)
-                {
-                    m_queueNameChange -= value;
-                }
-            }
-        }
+        public event EventHandler<QueueNameChangedAfterRecoveryEventArgs> QueueNameChangeAfterRecovery;
 
         public string ClientProvidedName { get; private set; }
 
-        public ushort ChannelMax
-        {
-            get { return m_delegate.ChannelMax; }
-        }
+        public ushort ChannelMax => m_delegate.ChannelMax;
```

Comment:
This is just me letting Resharper do some code reformatting. In retrospect it does add noise to the change and I could have skipped it.

Reply:
At the very least, you should make these sort of changes in a separate commit.

Reply:
At the very least, it's a good idea to make these type of changes in a separate commit.